hooks/pre-commit

```diff
@@ -1,9 +1,42 @@
-#! /bin/bash
+#!/bin/bash
```

Comment:
It would certainly have helped if I hadn't thinkoed when writing the commit comment/PR description. I have updated the PR description and improved a comment in 77ca493; is it better now?